src/ralph/models/edx/converters/xapi/base.py

```diff
@@ -0,0 +1,27 @@
+"""Base xAPI Converter"""
+
+import re
+
+from ralph.models.converter import BaseConverter
+
+
+class BaseXapiConverter(BaseConverter):
+    """Base xAPI Converter."""
+
+    def __init__(self, uuid_namespace, platform):
+        """Initializes BaseXapiConverter."""
+
+        self.platform = platform
+        super().__init__(uuid_namespace)
+
+    @staticmethod
+    def parse_course_id(course_id: str):
+        """Returns a dictionary with `course` and `module` of edX event's `context.course_id`."""
+
+        course_and_module = {"course": None, "module": None}
+        if not course_id:
+            return course_and_module
+        match = re.match(r"^course-v1:.+\+(.+)\+(.+)$", course_id)
+        if not match:
+            return course_and_module
+        return {"course": match.group(1), "module": match.group(2)}
```

Comment:
Maybe it can be made in a more compact way. 
```suggestion
        match = re.match(r"^course-v1:.+\+(.+)\+(.+)$", course_id)
        if match:
            return {"course": match.group(1), "module": match.group(2)}
        else:
            return {"course": None, "module": None}
```

Reply:
Good Idea) Thanks) Indeed, course_id is never `None`, removed the check)